src/App.tsx

```diff
@@ -1,10 +1,76 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { TypeError } from './types/TypeError';
 import { UserWarning } from './UserWarning';
+import { TodoAppHeader } from './components/TodoAppHeader';
+import { TodoAppTodo } from './components/TodoAppTodo';
+import { getTodos, deleteTodo } from './api/todos';
+import { TodoAppFooter } from './components/TodoAppFooter';
 
-const USER_ID = 0;
+const USER_ID = 6270;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [typeError, setTypeError] = useState('');
+  const [textError, setTextError] = useState('');
+  const [timerIdError, setTimerIdError] = useState<NodeJS.Timeout | null>(null);
+  const [filterByStatus, setFilterByStatus] = useState(Status.All);
+
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const areAllTodosCompleted = todos.every(todo => todo.completed);
+  let visibleTodos;
+
+  const getErrorText = () => {
+    return typeError === 'unexpected'
+      ? 'Unexpected error'
+      : `Unable to ${textError} a todo`;
+  };
+
+  const deleteHandler = () => {
+    setTypeError('');
+
+    deleteTodo()
+      .catch(() => {
+        setTypeError(TypeError.Delete);
+        setTextError(TypeError.Delete);
+      });
+  };
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(loadedTodos => setTodos(loadedTodos))
+      .catch(() => {
+        setTypeError(TypeError.Unexpected);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (timerIdError) {
+      clearTimeout(timerIdError);
```

Comment:
`timerId` is not a thing that should live in the state

to clear the timeout, you may use `timerId` variable saved in `const`. Do it inside `useEffect` returned function (check Effect cleanup block [here](https://www.w3schools.com/react/react_useeffect.asp))